metrics/option.go

```diff
@@ -0,0 +1,45 @@
+package metrics
+
+import (
+	"go.uber.org/zap"
+)
+
+// Option Server option
+type Option func(s *Server)
```

Comment:
Hi @nicholascapo would you mind adding a few test cases? The prometheus package is more tricky to test but option.go can be tested like https://github.com/cshum/imagor/blob/master/vips/option_test.go

Reply:
Thanks for this suggestion, while writing these tests I realized that I didn't want/need `WithAddr()` and I wanted to rename `Address` to `Host` for clarity, let me know what you think.